.github/ISSUE_TEMPLATE/upgrade.md

```diff
@@ -12,30 +12,34 @@ _period: 14 days
   - [ ] Bump internal version
   - [ ] Build and test new image locally with Azul's `make format`
   - [ ] Remove unused dependencies with high or critical CVEs
-  - [ ] Push commit to GitHub (directly to master branch, no PR needed)
+  - [ ] Push commit to GitHub (directly to `master` branch, no PR needed)
   - [ ] GH Action workflow succeeded
   - [ ] Image is available on [DockerHub](https://hub.docker.com/repository/docker/ucscgi/azul-pycharm) 
 - [ ] Update [Elasticsearch image](https://github.com/DataBiosphere/azul-docker-elasticsearch)
   - [ ] Bump [base image](https://hub.docker.com/_/elasticsearch/tags) tag (only minor and patch versions), if possible
   - [ ] Bump internal version 
   - [ ] Build and test new image locally with Azul's `make test`
   - [ ] Remove unused dependencies with high or critical CVEs
-  - [ ] Push commit to GitHub (directly to master branch, no PR needed)
+  - [ ] Push commit to GitHub (directly to `main` branch, no PR needed)
   - [ ] GH Action workflow succeeded
   - [ ] Image is available on [DockerHub](https://hub.docker.com/repository/docker/ucscgi/azul-elasticsearch) 
+- [ ] Update [BigQuery Emulator image](https://github.com/DataBiosphere/azul-bigquery-emulator)
+  - [ ] Bump [base image](https://hub.docker.com/_/debian/tags) tag, if possible
+  - [ ] Bump internal version 
+  - [ ] Push commit to GitHub (directly to `azul` branch, no PR needed)
+  - [ ] GH Action workflow succeeded
+  - [ ] Image is available on [DockerHub](https://hub.docker.com/r/ucscgi/azul-bigquery-emulator) 
```

Comment:
We need a testing strategy for this image, just like for the other two images. I'd be OK with simplifying the strategy in that it could use the image from DockerHub instead of the one built locally, but it should be the same strategy for all three images.

Reply:
@hannes-ucsc: "We determined that we don't actually need to manually test updated images for BQ Emulator, PyCharm, or Elasticsearch because those images are used directly from DockerHub during the GitHub Actions run for the PR branch that introduces the updates to the Azul codebase."